src/telegram.js

```diff
@@ -41,6 +41,7 @@ class TelegramBot extends EventEmitter {
    * @param {Boolean|Object} [options.webHook=false] Set true to enable WebHook or set options
    * @param {String} [options.webHook.key] PEM private key to webHook server.
    * @param {String} [options.webHook.cert] PEM certificate (public) to webHook server.
+   * @param {Boolean} [options.onlyFirstMatch=false] false: try matching all regexps; true: stop after first match
```

Comment:
A better description would be sth along the lines of "If true, stop after first match. Otherwise, all regexps are executed".